### PR TITLE
Update pty.pyi

### PR DESCRIPTION
**Pull request by sobolevn**
Source: https://github.com/python/cpython/blob/main/Lib/pty.py

### CODE REVIEW

File: stdlib/pty.pyi [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
-    STDOUT_FILENO: int
-    STDERR_FILENO: int
+    STDIN_FILENO: Literal[0]
+    STDOUT_FILENO: Literal[1]
+    STDERR_FILENO: Literal[2]
 
-    CHILD: int
+    CHILD: Literal[0]
     def openpty() -> tuple[int, int]: ...
     def master_open() -> tuple[int, str]: ...
```
Comment (AlexWaygood) [inline stdlib/pty.pyi:15]:
```suggestion
    def master_open() -> tuple[int, str]: ...  # deprecated, use openpty()
```

File: stdlib/pty.pyi [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
     def slave_open(tty_name: str) -> int: ...
```
Comment (AlexWaygood) [inline stdlib/pty.pyi:16]:
```suggestion
    def slave_open(tty_name: str) -> int: ...  # deprecated, use openpty()
```

**Comment by AlexWaygood** [comment]
^Suggestions are unrelated to the changes you're making, but we might as well, while we're here!

**Comment by sobolevn** [comment]
Thanks, important notice!

**Review by Akuli — APPROVED** [review]

**Pull request merged**